bullet_train/lib/bullet_train/resolver.rb

```diff
@@ -14,10 +14,17 @@ def run(eject: false, open: false, force: false, interactive: false)
       source_file[:relative_path] = nil
 
       if source_file[:absolute_path]
-        puts ""
-        puts "Absolute path:".green
-        puts "  #{source_file[:absolute_path]}".green
-        puts ""
+        if File.exist?(source_file[:absolute_path])
+          puts "" # standard:disable Style/IdenticalConditionalBranches
+          puts "Absolute path:".green
+          puts "  #{source_file[:absolute_path]}".green
+          puts ""
+        else
+          puts "" # standard:disable Style/IdenticalConditionalBranches
+          puts "Sorry, we could not find the file you're looking for.".red
+          puts "Please check the path and try again.".red
+          exit 1
```

Comment:
Ah-ha! It was the bare `exit` that was causing the swallowing of the test failures, and `exit 1` causes it to be passed through.